cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -21,11 +20,16 @@
    ========================================================================== #}
 
 {% macro render(items, active_item_id, additional_classes) %}
-
-<nav class="nav-secondary expandable {{ additional_classes if additional_classes else '' }}"
+{% from 'molecules/nav-link.html' import nav_link as nav_link %}
+<nav class="expandable
+            o-secondary-navigation
+            {{ additional_classes if additional_classes else '' }}"
```

Comment:
I wasn't sure if we were relying on this anywhere, but after I can't find any place it was being used so I'll remove it.